src/components/file/file.tsx

```diff
@@ -133,17 +133,13 @@ export class File {
     private chipSet;
     private mdcTextField;
 
-    public connectedCallback() {
-        this.initialize();
-    }
-
     public componentDidLoad() {
         this.chipSet = this.element.shadowRoot.querySelector(CHIP_SET_TAG_NAME);
         this.initialize();
     }
 
     private initialize() {
-        if (!this.chipSet) {
+        if (!this.mdcTextField) {
             return;
         }
```

Comment:
I don't understand how this can ever initialise the text field? `this.mdcTextField` is undefined until the following has been run, isn't it?

```ts
        this.mdcTextField = new MDCTextField(
            this.chipSet.shadowRoot.querySelector('.mdc-text-field'),
        );
```

And that won't ever be run, because the function will exit here…?

Unless something else has changes so that `this.mdcTextField` is set somewhere else? I'm puzzled how this can work at all 😅 Maybe we no longer need the `MDCTextField` functionality at all?

But, if nothing relating to this has changed in this PR, I think it should be addressed in a separate PR.

Reply:
This is my mistake, my intention was to write `if (this.mdcTextField)` 😅

Reply:
I actually removed the usage of MDCTextField altogether. I didn't think it made much sense and when I tested it it actually behaved better with showing invalid states. Maybe I'm missing something crucial here 😅
See https://github.com/Lundalogik/lime-elements/pull/2820/commits/79e0db9dff4ce1994108055e08b6510f6bdd4f29